### DISCUSSION TITLE
How do you override the visited pseudo class with another pseudo class?

### DISCUSSION THREAD

**Discussion by neilchaudhuri** [chakra-ui/chakra-ui#1998] [General] [answered]
I have a `Link` with `_hover` and `_visited` CSS configuration, but I have been unable to figure out how to have `_hover` apply on a `_visited` link.

In other words I want to do in Chakra UI what in traditional CSS one might do like this:

`a:visited:hover {...}`

Any ideas? Thanks.

**Comment by with-heart** [comment, marked as answer]
Hi @neilchaudhuri! Unfortunately we don't currently have a built-in way to target `:visited:hover`. You can instead use the `sx` prop to handle this: https://codesandbox.io/s/exciting-euler-exro0?file=/src/App.tsx

**Reply by neilchaudhuri** [reply]
Thanks for the pointer. We are currently on `0.7.0`. Is `sx` available on that version or in later versions? We have been planning on an upgrade, but this may accelerate the timeline

**Reply by with-heart** [reply]
You can use the `jsx` pragma from `@emotion/core` to use the `css` prop on any component: https://codesandbox.io/s/strange-goldstine-orjfv?file=/src/App.tsx

The downside to this is that it's not connected to the Chakra theme, so you have to read the `theme` directly rather than using Chakra props like `blue.500`.
